### PR TITLE
Nr artillery setup

### PR DESCRIPTION
**Pull request by tyler-thee-creator**
Trello ticket this is for: https://trello.com/c/3w0mAJPn/32-implement-new-relic-artillery-with-service

**PR for setting up new relic and artillery w/ service**

Some files may have small changes but you can focus on:
- stress_testing (folder)

### CODE REVIEW

File: stress_testing/functions.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { featuresTableRecord } = require('../database/data_seeding/fake_data_records.js');
+
+function setUrlWithProductId(requestParams, context, ee, next) {
+  const bottomTenPercentId = 9000000 + Math.ceil(Math.random() * 1000000);
```
Comment (masoncpott) [inline stress_testing/functions.js:4]: nicely done figuring this out.!

File: stress_testing/get.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - duration: 60
+      arrivalRate: 10
+      name: low
+    - duration: 60
+      arrivalRate: 100
+      name: medium
+    - duration: 60
+      arrivalRate: 1000
+      name: high
+  processor: './functions.js'
```
Comment (masoncpott) [inline stress_testing/get.yml:16]: I am not sure about this but you might have to apply the processor to each phase...? If you conducted the test and this worked, then obviously you dont have to but I have seen it before where the processor is applied for each phase duration.

**Review by masoncpott — APPROVED** [review]
This is a great start to setting up testing. I think you should add the newrelic_agent.log to your gitignore just to be safe. Is it necessary to have separate files for a GET test and a POST test? You could have both of those in one file that test GET and POST. However, if you wanted to isolate a bottleneck, it might be easier to separate those two routes like you have now. Just some thoughts that came to mind.

File: stress_testing/get.yml (hunk 2 above)
Comment (tyler-thee-creator) [reply stress_testing/get.yml:16]: good point, i'll def look into this and adjust if need be

**Pull request merged**